rules/show-certification-award/rule.md

```diff
@@ -0,0 +1,39 @@
+---
+type: rule
+title: Do you show Certification Reward?
+uri: show-certification-award
+authors:
+  - title: Stephan Fako
+    url: https://www.ssw.com.au/people/stephan-fako/
+related: []
+created: 2024-04-08T06:55:48.587Z
+guid: 94ab00f4-a6d5-4e8f-aca3-68d6cc0a3498
+---
+**Do you show Certification Reward?**
```

Comment:
Hi @stephanfakossw 

You shouldn't repeat the title here in the content

- [ ] 1. Please remove